js/scripts.js

```diff
@@ -239,30 +239,71 @@ window.onload = () => {
 
   // AddPortfolioWork(): Called of the Function to fill the PortfolioElement.
   addPortfolioWork();
-};
 
-/* CONTACTS FORM VALIDATION */
-function isUpperCase(letter) {
-  const REGEX = /^[A-Za-z]+$/;
-  return !!(letter === letter.toUpperCase() && REGEX.test(letter));
-}
+  /* CONTACTS FORM VALIDATION */
+  function isUpperCase(letter) {
+    const REGEX = /^[A-Za-z]+$/;
+    return !!(letter === letter.toUpperCase() && REGEX.test(letter));
+  }
+
+  const submitError = document.querySelector('#submit__error');
+  const formElement = document.querySelector('#form');
 
-const submitError = document.querySelector('#submit__error');
-const formElement = document.querySelector('#form');
+  formElement.addEventListener('submit', (event) => {
+    const emailValue = event.target[1].value;
+    const emailArray = emailValue.split('');
 
-formElement.addEventListener('submit', (event) => {
-  const emailValue = event.target[1].value;
-  const emailArray = emailValue.split('');
+    for (let index = 0; index < emailArray.length; index += 1) {
+      const letter = emailArray[index];
+      const validator = isUpperCase(letter);
+      if (validator) {
+        submitError.style.visibility = 'visible';
+        event.preventDefault();
+        break;
+      } else {
+        submitError.style.visibility = 'hidden';
+      }
+    }
+  });
 
-  for (let index = 0; index < emailArray.length; index += 1) {
-    const letter = emailArray[index];
-    const validator = isUpperCase(letter);
-    if (validator) {
-      submitError.style.visibility = 'visible';
-      event.preventDefault();
-      break;
-    } else {
-      submitError.style.visibility = 'hidden';
+  // Function: InjectFormData()
+  // Put the information contained within the LocalStorage item called userData if the object exist.
+  // userData contains name, email and message.
+  const inputFields = document.querySelectorAll('.input');
+  function injectFormData() {
+    const object = JSON.parse(localStorage.getItem('userData'));
+    if (object) {
+      inputFields.forEach((input) => {
+        const value = object[input.id];
+        input.value = value;
+      });
```

Comment:
[OPTIONAL]
- [ ] I kindly suggest that you make use of the example code implementation below to upate the form fields when the user reload the webpage :+1: :sparkles: 
- > EXAMPLE CODE
```javascript
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};

```